nginx/README-QUIC.md

```diff
@@ -0,0 +1,56 @@
+# NGINX with OQS-BoringSSL for QUIC
+
+This Docker setup provides an nginx instance configured to use OQS-BoringSSL, which supports QUIC with quantum-safe algorithms. For more information on the supported quantum-safe algorithms and how to enable additional algorithms, please refer to the following resources:
+
+- [Supported Algorithms](https://github.com/open-quantum-safe/boringssl?tab=readme-ov-file#supported-algorithms)
+- [Using LibOQS Algorithms Not in the Fork](https://github.com/open-quantum-safe/boringssl/wiki/Using-liboqs-algorithms-not-in-the-fork)
+
+## Setup Instructions
+
+### Step 1: Build the Docker Image
+
+Build the Docker image using the provided Dockerfile:
+
+```bash
+docker build -f Dockerfile-QUIC .
+```
+
+After building, remember the SHA256 hash of the image from the last line of the output.
```

Comment:
This looks complicated: Why not built to a name (and reference that below)?

Reply:
updated